example/src/test/java/uk/gov/moj/cpp/jobmanager/example/util/OpenEjbConfigurationBuilder.java

```diff
@@ -0,0 +1,88 @@
+package uk.gov.moj.cpp.jobmanager.example.util;
+
+
+
+import java.util.Properties;
+
+
+public class OpenEjbConfigurationBuilder {
```

Comment:
We have an OpenEjbConfigurationBuilder in test-utils-core. If that doesn't give you all you need, update that one rather then create a new one

Reply:
This will have to be done later together with the move to Postgres only discussed in other comments below